<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="kalyansai" name="Yahoo" tests="5" failures="1" timestamp="18 Jun 2016 16:34:09 GMT" time="0.005" errors="0">
  <testcase name="YahooLogin" time="0.0" classname="Test1.YahooTest"/>
  <testcase name="YahooLogin" time="0.0" classname="Test1.YahooTest"/>
  <testcase name="testRecieveMail" time="0.001" classname="Test1.YahooTest"/>
  <testcase name="testSendingMail" time="0.002" classname="Test1.YahooTest">
    <failure type="java.lang.AssertionError" message="expected [Bad] but found [Good]">
      <![CDATA[java.lang.AssertionError: expected [Bad] but found [Good]
	at Test1.YahooTest.testSendingMail(YahooTest.java:22)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- testSendingMail -->
  <testcase name="YahooLogout" time="0.0" classname="Test1.YahooTest"/>
</testsuite> <!-- Yahoo -->
